cooperatives.yml

```diff
@@ -254,6 +254,14 @@ librecode-gitlab:
   description: "LibreCode - Open source technology solutions"
   location: Brazil
 
+libresign:
+  source: github
+  login: libresign
+  name: LibreCode Coop
```

Comment:
Let's use "LibreSign" or similar here in order to be listed different that the already existing LibreCode Coop

Reply:
But is the same coop. We created a new organization at GitHub only to maintain all repositories of LibreSign together.

Reply:
I know, but in order to be listed different here https://coophub.io/orgs/

Reply:
Sorry, I didn't understood as well what you want.

Now, with the name `LibreCode Coop` have two organizations, one at GitHub and other at GitLab. With this change wouldn't just appear one more `LibreCode Coop`?

![Screenshot_20230502_111551](https://user-images.githubusercontent.com/1079143/235693389-56db7720-0fdb-44ef-b9c0-b001342e3d0b.png)

Reply:
Fala galera!
No problem, we currently have those 2, because for now we treat them as different orgs. But merging this PR we will have 3 if you don't change the name. For that reason I'm suggesting at least changing it to LibreSign (you could mention it belongs to LibreCode in the description)

Reply:
Yes sure, that's something nice to have... it's not a trivial change

Reply:
Yes, I know. Will be necessary refactor the place that render this page.

Maybe will be necessary to change the yaml structure to don't implement a workaround but to implement a good change.

But, I think that will be best to have this discussion in a issue, not at this PR.

And, about your request, what you want that me do?

Reply:
Yes, thank you, feel free to create an issue and let's see if there's time at Fiqus or external people to do that.
Regarding this PR, I'd do what I suggested here:

> But merging this PR we will have 3 if you don't change the name. For that reason I'm suggesting at least changing it to LibreSign (you could mention it belongs to LibreCode in the description)

Reply:
@pbrudnick could you check again if is this that you want?

Reply:
I also created this issue #139 as follow up of our discussion.

Reply:
Yes, obrigado Vitor!

Reply:
Disponha!